products/ocp4/profiles/pci-dss.profile

```diff
@@ -20,6 +20,7 @@ filter_rules: '"ocp4-node" not in platforms and "ocp4-master-node" not in platfo
 
 # Req-2.2
 extends: cis
+extends: pci-dss-3.2.1
 
 selections:
```

Comment:
Remove these selections since they're going to be inherited from the extended profile.